products/memgraph/upcoming/tutorials/football-transfers.md

```diff
@@ -0,0 +1,390 @@
+WHERE
+    team.name = "FC Barcelona"
+WITH DISTINCT player
+RETURN player.position as player_position, COUNT(player) AS position_count, collect(player.name) as player_names
+ORDER BY position_count DESC
+```
+
+5) Football has seen a lot of rivalries develop between clubs during its rich and long history.
+One of the most famous ones is between fierce rivals FC Barcelona and Real Madrid.
+There is a term, El Clasico, for a match between those two teams. Let's find all the transfers between
+FC Barcelona and Real Madrid.
+
+```opencypher
+MATCH
+    (m:Team)-[:TRANSFERRED_FROM]-(t:Transfer)-[:TRANSFERRED_TO]-(n:Team),
+    (t)<-[:TRANSFERRED_IN]-(p:Player)
+WHERE
+    (m.name = "FC Barcelona" AND n.name = "Real Madrid") OR
+    (m.name = "Real Madrid" AND n.name = "FC Barcelona")
+RETURN m.name as transferred_from_team, p.name as player_name, n.name as transfered_to_team
+```
+
+6) FC Barcelona is one of the most valuable football clubs in the world. Players often want to play there as long as possible.
+But what about those players who didn't fit in well? Where do they go?
+
+```opencypher
+MATCH
+    (m:Team)-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(p:Player),
+    (t)-[:TRANSFERRED_TO]->(n:Team)
+WHERE
+    m.name = "FC Barcelona"
+RETURN n.name as team_name, collect(p.name) as player_names, COUNT(p) AS number_of_players
+ORDER BY number_of_players DESC
+```
+
+7) What are the teams that most players went to in season 2003/2004? The results may surprise you.
+
+```opencypher
+MATCH
+    (season:Season)<-[:HAPPENED_IN]-(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player),
+    (t)-[:TRANSFERRED_TO]->(team:Team)
+WHERE
+    season.name = "2003/2004"
+WITH DISTINCT player, team
+RETURN team.name as team_name, COUNT(player) AS number_of_players, collect(player.name) as player_names
+ORDER BY number_of_players DESC, team_name
+LIMIT 20;
+```
+
+8) In great teams, there are players who seem to be irreplaceable. When they leave,
+the club board is often struggling to find a proper replacement for them. Let's find out which positions
+club "FC Barcelona" spent money on in season 2015/2016.
+
+```opencypher
+MATCH
+    (:Team)-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player),
+    (s:Season)<-[:HAPPENED_IN]-(t)-[:TRANSFERRED_TO]->(m:Team)
+WHERE
+    t.fee IS NOT NULL AND
+    s.name = "2015/2016" AND
+    m.name = "FC Barcelona"
+RETURN collect(player.name) AS player_names, player.position AS player_position, ROUND(SUM(t.fee)) + 'M €' AS money_spent_per_position
+```
+
+9) But what was the highest transfer amount per position FC Barcelona spent on in seasons from 1992/1993 till 2019/2020?
+
+```opencypher
+MATCH
+    (:Team)-[:TRANSFERRED_FROM]->(t:Transfer)<-[:TRANSFERRED_IN]-(player:Player),
+    (t)-[:TRANSFERRED_TO]->(team:Team)
+WHERE
+    t.fee IS NOT NULL AND
+    team.name = "FC Barcelona"
+RETURN MAX(t.fee) + 'M €' AS max_money_spent, player.position as player_position
```

Comment:
This query doesn't have ordering. I suppose that the results should be ordered by the sum?

Reply:
:+1: